pageserver/src/http/routes.rs

```diff
@@ -2307,6 +2308,38 @@ async fn post_tracing_event_handler(
     json_response(StatusCode::OK, ())
 }
 
+async fn force_aux_policy_switch_handler(
+    mut r: Request<Body>,
+    _cancel: CancellationToken,
+) -> Result<Response<Body>, ApiError> {
+    check_permission(&r, None)?;
+    let tenant_shard_id: TenantShardId = parse_request_param(&r, "tenant_shard_id")?;
+    let timeline_id: TimelineId = parse_request_param(&r, "timeline_id")?;
+    let policy: AuxFilePolicy = json_request(&mut r).await?;
+
+    let state = get_state(&r);
+
+    let process = || async move {
+        let tenant = state
+            .tenant_manager
+            .get_attached_tenant_shard(tenant_shard_id)?;
+        tenant.wait_to_become_active(ACTIVE_TENANT_TIMEOUT).await?;
+        let timeline =
+            active_timeline_of_active_tenant(&state.tenant_manager, tenant_shard_id, timeline_id)
+                .await?;
+        timeline.force_switch_aux_policy(policy)?;
+        Ok(())
+    };
+
+    match process().await {
+        Ok(st) => json_response(StatusCode::OK, st),
+        Err(err) => json_response(
+            StatusCode::INTERNAL_SERVER_ERROR,
```

Comment:
It's not very important for this temporary API, but in general one should avoid forcing all the possible errors into 500s -- things like `wait_to_become_active` return error types that know how to return 503, timeout error codes etc, so should be allowed to convert into ApiError via their `?`